### PR TITLE
Separate CLI project code to different repository

### PR DESCRIPTION
**Pull request by xeho91**
This is just a cleanup, and moving code files for Lilly CLI projects into a spearate repository:
https://github.com/TeamArmadilloOrg/lilly-cli

I reconfigured some of the config files, such as webpack. Hopefully I haven't forgotten about anything.

### CODE REVIEW

File: config/.prettierignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.vscode
+build
+binary
```
Comment (mariushilleke) [inline config/.prettierignore:3]: The `/binary`-directory is deleted as well, right? If yes, the entry for that directory could be deleted, as well.

Comment (xeho91) [reply config/.prettierignore:3]: Indeed, thanks

**Comment by xeho91** [comment]
> 
> 
> This looks like you were able to remove the necessary files. I am still not up to speed on all of this, but will work to understand so I can better contribute.

Feel free to reach me, if there's something to explain. I'd love to help you understand faster.

File: config/.prettierignore (hunk 1 above)
Comment (xeho91) [reply config/.prettierignore]: Resolved in last commits (force pushed)

**Pull request merged**